Sources/Filters.swift

```diff
@@ -123,7 +129,11 @@ struct StringFilters {
     case "true", "yes", "1":
       return true
     default:
-      return nil
+      if required {
```

Comment:
What I meant with my throwing behaviour was more along the lines of, if I pass the parameter `someInvalidStuff` instead of `true` or `false`, that should be considered and thus throw.

I'd change the guard to:
```swift
guard index < arguments.count, let boolArg = arguments[index] as? String else {
  ... required check...
```
In the switch, you then don't check for `required`, the default case always throws.